vulture/config.py

```diff
@@ -0,0 +1,212 @@
+"""
+This module handles retrieval of configuration values from either the
+command-line arguments or the pyproject.toml file.
+"""
+import argparse
+import sys
+from os.path import abspath, exists
+
+import toml
+
+from .version import __version__
+
+#: Possible configuration options and their respective defaults
+DEFAULTS = {
+    "min_confidence": 0,
+    "paths": [],
+    "exclude": [],
+    "ignore_decorators": [],
+    "ignore_names": [],
+    "make_whitelist": False,
+    "sort_by_size": False,
+    "verbose": False,
+}
+
+#: sentinel value to distinguish between "False" and "no default given"
+MISSING = object()
```

Comment:
Instead of having this global variable, we could just delete all keys with MISSING values from the CLI dict before returning it in _parse_args(). If you don't want to do this change, I can also do it after merging.